### ISSUE TITLE
multiple emoji reactions

### ISSUE THREAD

**Issue by breadguyyy** [juandjara/wafrn-rn#9]
allow clicking on multiple reactions before closing the picker

**Comment by juandjara** [comment]
I'm not sure if wafrn backend supports this @gabboman (multiple reactions on a post)

**Comment by gabboman** [comment]
yeah this is not misskey, we use akkoma style wich allow multiple emojireacts per post BUTT if reacting to mastodon it goes to the void [EDIT each emojireact would be one call to the emojireact endpoint]

**Comment by breadguyyy** [comment]
to the void for everybody? like no one can see it?

**Comment by gabboman** [comment]
> to the void for everybody? like no one can see it?

I mean mastodon does not process emojireacts hence they dont get displayed. If not sure what the person reciving an emojireact I recomend like + emojireact

**Comment by breadguyyy** [comment]
I mean can other wafrn/akkoma/misskey/glitch users still see emoji reacts
on mastodon posts?

**Comment by juandjara** [comment]
Yes, other users can see emoji reacts if their software supports it because the reacts are stored on wafrn

**Comment by breadguyyy** [comment]
ok that's what I thought lol gabbo had me nervous for a second
